infra/gcp/bash/ensure-prod-storage.sh

```diff
@@ -391,6 +391,28 @@ function ensure_all_prod_special_cases() {
     ensure_gcs_role_binding "gs://k8s-artifacts-gcslogs" \
         "group:k8s-infra-gcs-access-logs@kubernetes.io" \
         "objectViewer"
+
+    # Special case: In order to run the container image signing tests, k8s-cip-test-prod
+    # requires to have the IAM Service Account Credentials API enabled to generate OIDC
+    # identity tokens. The production project needs the API too to sign all promoted images.
+
+    color 6 "Ensuring IAM Service Account Credentials API for image signing"
+    ensure_services "${IMAGE_PROMOTER_TEST_PROD_PROJECT}" "iamcredentials.googleapis.com"
+    ensure_services "${PROD_PROJECT}" "iamcredentials.googleapis.com"
```

Comment:
Why enable this service for `k8s-artifacts-prod` ? do we need to get tokens after promotion is done ?

Reply:
Correct! Once images are copied to the target registries, the promoter will need to generate tokens with the identity of the Main Kubernetes Signer. I've opened an issue to better visualize the flows. This PR is part of the Promotion and e2e flows:

https://github.com/kubernetes/release/issues/2445